app/assets/javascripts/proposals.coffee

```diff
@@ -21,11 +32,25 @@ $(document).ready ->
           error.insertAfter(element)
       rules:
         'rejection[reasons]':
-          required: true
+          required:
+            depends:
+              -> $('#rejection-note').val()
         'rejection[note]':
-          required: true
+          required: 
+            depends:
+              -> $('#proposal-rejection-reasons').val().length > 0
       messages:
         'rejection[reasons][]':
           required: 'Reason(s) are required'
         'rejection[note]':
           required: 'Note is required'
+          
+    updateRequiredIcons = -> 
```

Comment:
No need to do anything now, but just thinking that it seems like we do this sort of thing quite a bit with many of the other forms, and we might want to try and make a generic method for simpler fields like these.